dubbo-remoting/dubbo-remoting-api/src/main/java/org/apache/dubbo/remoting/telnet/support/TelnetHandlerAdapter.java

```diff
@@ -24,6 +24,7 @@
 import org.apache.dubbo.remoting.transport.ChannelHandlerAdapter;
 
 public class TelnetHandlerAdapter extends ChannelHandlerAdapter implements TelnetHandler {
+    private static volatile boolean enableTelnet = true;

```

Comment:
It is not a good way to close telnet.
This property should be bound to URL.

Reply:
you are right